README.md

```diff
@@ -0,0 +1,33 @@
+## Covid-Stats
+
+> This is a React.js app that displays Covid-19 data organized per country. Al data is updated daily and comes from the narrativa.com APIs
+
+## Built With
+
+- React
+- Redux
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository by running git clone this repository.
+- Navigate to the folder 
+- Install packages by running in the terminal `npm i`
+- Run the live server with the command `npm start`
```

Comment:
- [ ] As asked by  the previous reviewer, please add a live link using one of the above mentioned options